lib/auth/mfa/mfav1/service.go

```diff
@@ -16,18 +16,439 @@
 
 package mfav1
 
-import mfav1 "github.com/gravitational/teleport/api/gen/proto/go/teleport/mfa/v1"
+import (
+	"cmp"
+	"context"
+	"log/slog"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/api/client/proto"
+	mfav1 "github.com/gravitational/teleport/api/gen/proto/go/teleport/mfa/v1"
+	"github.com/gravitational/teleport/api/types"
+	apievents "github.com/gravitational/teleport/api/types/events"
+	wanlib "github.com/gravitational/teleport/lib/auth/webauthn"
+	wantypes "github.com/gravitational/teleport/lib/auth/webauthntypes"
+	"github.com/gravitational/teleport/lib/authz"
+	"github.com/gravitational/teleport/lib/events"
+)
+
+// AuthServer defines the subset of Auth server methods used by the MFA service.
```

Comment:
Suggestion: Be 100% specific.

```suggestion
// AuthServer defines the subset of lib/auth.Server methods used by the MFA service.
```